components/actionbutton/stories/template.js

```diff
@@ -128,3 +128,75 @@ export const Template = ({
 		</button>
 	`;
 };
+
+export const ActionButtonsWithIconOptions = ({
+	...args
+}, context ) => Container({
+	withBorder: false,
+	direction: "row",
+	wrapperStyles: {
+		columnGap: "12px",
+	},
+	content: html`
+    ${Template({
+      ...args,
+    }, context )}
+    ${Template({
+      ...args,
+    }, context )}
+    ${Template({
+      ...args,
+      hideLabel: true,
+    }, context )}
+	${Template({
+      ...args,
+      hideLabel: true,
+      hasPopup: "true",
+    }, context )}
+  `,
+}, context );
+
+export const IconOnlyOption = ({
+	...args
+}, context ) => Container({
+	withBorder: false,
+	direction: "row",
+	wrapperStyles: {
+		columnGap: "12px",
+	},
+	content: html`
+	${Template({
+      ...args,
+      hideLabel: true,
+      hasPopup: "true",
+    }, context )}
+	${Template({
+      ...args,
+      hideLabel: true,
+			isQuiet: true,
+      hasPopup: "true",
+    }, context )}
+	`
+});
```

Comment:
The indentation needs some adjustment in this function.

Reply:
I believe there's some indentation fixes needed in `ActionButtonsWithIconOptions` as well.

Reply:
I'm still seeing some indentation issues [here](https://github.com/adobe/spectrum-css/blob/666230f1b7c151398d2db0ec399637f3e31b9d54/components/actionbutton/stories/template.js#L132-L184). I think it's just another case of there being mixed tabs and spaces, where converting all to tabs should help with adjusting.

Reply:
➕ 1️⃣